src/Dapr.Actors/Client/ActorProxy.cs

```diff
@@ -76,7 +84,7 @@ public static TActorInterface Create<TActorInterface>(ActorId actorId, string ac
         public static object Create(ActorId actorId, Type actorInterfaceType, string actorType)
         {
             _ = actorInterfaceType as IActor ?? throw new ArgumentException("The interface must implement IActor.", nameof(actorInterfaceType));
-            return DefaultProxyFactory.CreateActorProxy(actorId, actorInterfaceType, actorType);
+            return ActorProxyFactory.CreateActorProxy(actorId, actorInterfaceType, actorType);
```

Comment:
Do we have a test that calls this method? Line 86 will always fail....

Reply:
You are correct - I just noticed this issue too. Once we decide how to do the configuration I'll make sure to get tests made for the new methods.

Reply:
This is true - looking at the git comments for this code I believe I fixed what it was intending to do w/: 
`typeof(IActor).IsAssignableFrom(actorInterfaceType)`

I'll get a unit test to hit it to verify.